kernel/src/common/argparse.c

```diff
@@ -83,10 +83,10 @@ static void argparse_parse(char *cmdline)
 		{
 			os_printf("RUNNING TESTS\n");
 			os_printf("Running tests...\n");
-			Test *tests[2];
-			tests[0] = create_test("This passes", &test1);
-			tests[1] = create_test("This fails", &test2);
-			run_tests(tests, 2);
+//			Test *tests[2];
+//			tests[0] = create_test("This passes", &test1);
+//			tests[1] = create_test("This fails", &test2);
+//			run_tests(tests, 2);
```

Comment:
Do these tests still work as it would be nice to add some actual ones

Reply:
working on it